src/request.rs

```diff
@@ -332,3 +332,14 @@ fn finish_request_context(request_id: Option<i64>, status_code: i32) -> crate::R
 
     Ok(())
 }
+
+pub fn skywalking_set_request_name(args: &mut [ZVal]) -> phper::Result<()> {
```

Comment:
`operation name` is the official term for the span. Why is this called `request name`? We don't have this concept.

Reply:
Renamed to skywalking_set_operation_name